supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesWorkflowPaths.scala

```diff
@@ -13,22 +17,36 @@ object JesWorkflowPaths {
   def apply(workflowDescriptor: BackendWorkflowDescriptor, jesConfiguration: JesConfiguration, gcsFileSystem: GcsFileSystem) = {
     new JesWorkflowPaths(workflowDescriptor, jesConfiguration, gcsFileSystem)
   }
-}
 
-class JesWorkflowPaths(workflowDescriptor: BackendWorkflowDescriptor, jesConfiguration: JesConfiguration, gcsFileSystem: GcsFileSystem) {
+  implicit class GoogleAuthWorkflowOptions(val workflowOptions: WorkflowOptions) extends AnyVal {
+    def toGoogleAuthOptions: GoogleAuthMode.GoogleAuthOptions = new GoogleAuthOptions {
+      override def get(key: String): Try[String] = workflowOptions.get(key)
+    }
+  }
+}
 
+class JesWorkflowPaths(workflowDescriptor: BackendWorkflowDescriptor, jesConfiguration: JesConfiguration, val gcsFileSystem: GcsFileSystem) {
```

Comment:
Maybe `gcsFileSystem` should get a more specific name to prevent the confusion that led to the original bug?  `gcsFileSystemWithUserAuth` or something?